pennylane/ops/qubit.py

```diff
@@ -407,6 +431,9 @@ def _matrix(cls, *params):
     def _eigvals(cls, *params):
         return cls.eigvals
 
+    def adjoint(self, do_queue=False):
+        return CZ(wires=self.wires, do_queue=do_queue)
```

Comment:
If it weren't so dangerous/silly I would almost suggest making all gates self-adjoint by default

Reply:
Uh, rather not :)

Reply:
I'm on the same boat as maria.

Reply:
Yes, agree 😆 Just struck me how many gates are just self-adjoint.